src/compressed_tensors/quantization/observers/base.py

```diff
@@ -22,9 +23,26 @@
 from compressed_tensors.registry.registry import RegistryMixin
 from torch import FloatTensor, IntTensor, Tensor
 from torch.nn import Module
+from enum import Enum
 
 
-__all__ = ["Observer"]
+_LOGGER = logging.getLogger(__name__)
+
+
+__all__ = ["Observer", "ObserverTypes"]
+
+class ObserverTypes(str, Enum):
```

Comment:
take a look at how the names are initially created dynamically - good idea to formalize references to the names, but wonder if there's a way we can keep some of the dynamic structure and also generalize for scales and zero points as well https://github.com/neuralmagic/compressed-tensors/blob/d914b734c2e5fc6ae15ffc26b80807c815b88444/src/compressed_tensors/quantization/lifecycle/initialize.py#L85C31-L85C40